swagger.yml

```diff
@@ -878,27 +816,40 @@ definitions:
         "timeline":
           [
             {
-              "eventName": "SHIPMENT_CREATED",
```

Comment:
Slightly changed the timeline content because:
1. The events is not what we would realistically see up to `OUT_FOR_DELIVERY`, because it did not included `CREATED` and `GECODED` events.
2. The IDs should alternate from OrderID and PackageTrackingNumbers. But before they seemed random.
3. Shortened the timeline for brevity